lib/exq/redis/script.ex

```diff
@@ -51,6 +51,38 @@ defmodule Exq.Redis.Script do
         return {1, conflict_jid}
       end
       """),
+    enqueue_all:
```

Comment:
Keys will come as:
```
[schedule_queue, job_1_unique_key, job_2_unique_key, ...]
```
And args will come as:
```elixir
[
  job_1_jid, job_1_score, job_1_serlialized_job, job_1_unlocks_in, # indexes 1, 2, 3, 4
  job_2_jid, job_2_score, job_2_serlialized_job, job_2_unlocks_in, # indexes 5, 6, 7, 8
 ...
]
```

Reply:
According to https://redis.io/docs/manual/programmability/eval-intro/ I think we do not even need MULTI/EXEC as scripts are already run atomically.

Reply:
@ananthakumaran please take a quick look when you can. Thanx :)